src/app/pages/OriginsLaunchpad/pages/SalesDay/pages/BuyStep/components/BuyDialog/components/BuySection/styled.ts

```diff
@@ -26,13 +26,14 @@ export const BuyButton = styled.button<IBuyButtonProps>`
   text-transform: uppercase;
   padding: initial;
 
-  &:hover {
+  &:not([disabled]):hover {
```

Comment:
I probably didn't notice it before but we don't use `styled components` in our code anymore, we use `CSS modules` instead. Rewrite this file to `index.module.scss` instead and you can take a look at some of ours `index.module.scss` for inspiration.

Also, rewrite all of the `styled.ts` files used in this PR to `CSS modules`.